src/App.jsx

```diff
@@ -14,70 +16,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const noGoodSelectedBlock = (
     <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+  );
 
+  const goodSelectedBlock = (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+      {good}
+      {' is selected'}
```

Comment:
unnecessary line break
```suggestion
      {good}  is selected
```

Reply:
> unnecessary line break
i made it because linter forced me to

Reply:
> unnecessary line break

also this is the reason why i can't just remove template string. I need a space after the good.